README.md

```diff
@@ -520,51 +520,76 @@ The following command line options are available
   -s    
   -skipVersionCheck
         Indicates the service should skip the Core Service's version compatibility check.
+  -n
+  --serviceNamey                
```

Comment:
Typo :)
Other than this LGTM. I'll change the longform from "name" to "serviceName" in CSDK.